core/events/workspace_events.ts

```diff
@@ -75,12 +67,18 @@ export class FinishedLoading extends AbstractEvent {
    *
    * @param json JSON representation.
    */
-  override fromJson(json: AnyDuringMigration) {
+  override fromJson(json: FinishedLoadingJson) {
     this.isBlank = false;
-    this.workspaceId = json['workspaceId'];
-    this.group = json['group'];
+    this.workspaceId = json['workspaceId'] || '';
+    this.group = json['group'] || '';
   }
 }
 
+export interface FinishedLoadingJson {
```

Comment:
why doesn't this one extend AbstractEventJson? since the class itself does extend AbstractEvent?

Reply:
Hmm I didn't extend it b/c `fromJson` and `toJson` don't call `super`, but I think they actually should? So I'll switch this to extend as well.